### PR TITLE
merge 4.3.0 to 5.0

### PR DESCRIPTION
**Pull request by z8674558**

### CODE REVIEW

File: .github/workflows/run_cts_tests.yaml [hunk 1]
```diff
@@ -78,10 +78,10 @@ jobs:
         if: matrix.connect_type == 'tls'
         run: |
           docker-compose -f .ci/compatibility_tests/docker-compose-mongo-tls.yaml up -d
-          sed -i 's|^[#[:space:]]*auth.mongo.ssl[[:space:]]*=.*|auth.mongo.ssl = on|g' apps/emqx_auth_mongo/etc/emqx_auth_mongo.conf
-          sed -i 's|^[#[:space:]]*auth.mongo.cacertfile[[:space:]]*=.*|auth.mongo.cacertfile = /emqx/apps/emqx_auth_mongo/test/emqx_auth_mongo_SUITE_data/ca.pem|g' apps/emqx_auth_mongo/etc/emqx_auth_mongo.conf
-          sed -i 's|^[#[:space:]]*auth.mongo.certfile[[:space:]]*=.*|auth.mongo.certfile = /emqx/apps/emqx_auth_mongo/test/emqx_auth_mongo_SUITE_data/client-cert.pem|g' apps/emqx_auth_mongo/etc/emqx_auth_mongo.conf
-          sed -i 's|^[#[:space:]]*auth.mongo.keyfile[[:space:]]*=.*|auth.mongo.keyfile = /emqx/apps/emqx_auth_mongo/test/emqx_auth_mongo_SUITE_data/client-key.pem|g' apps/emqx_auth_mongo/etc/emqx_auth_mongo.conf
+          sed -i 's|^[#[:space:]]*auth.mongo.ssl[[:space:]]*=.*|auth.mongo.ssl.enable = on|g' apps/emqx_auth_mongo/etc/emqx_auth_mongo.conf
```
Comment (zmstone) [inline .github/workflows/run_cts_tests.yaml]: missing `.enable` on the left side of `=` ?

File: .github/workflows/run_cts_tests.yaml [hunk 2]
```diff
@@ -132,7 +132,7 @@ jobs:
         if: matrix.connect_type == 'tls'
         run: |
           docker-compose -f .ci/compatibility_tests/docker-compose-mysql-tls.yaml up -d
-          sed -i 's|^[#[:space:]]*auth.mysql.ssl[[:space:]]*=.*|auth.mysql.ssl = on|g' apps/emqx_auth_mysql/etc/emqx_auth_mysql.conf
+          sed -i 's|^[#[:space:]]*auth.mysql.ssl[[:space:]]*=.*|auth.mysql.ssl.enable = on|g' apps/emqx_auth_mysql/etc/emqx_auth_mysql.conf
```
Comment (zmstone) [inline .github/workflows/run_cts_tests.yaml:135]: missing `.enable` on the left side of `=` ?

File: apps/emqx_auth_pgsql/etc/emqx_auth_pgsql.conf [hunk 3]
```diff
@@ -6,8 +6,8 @@
 ##
 ## Value: Port | IP:Port
 ##
-## Examples: 5432, 127.0.0.1:5432, localhost:5432
-auth.pgsql.server = 127.0.0.1:5432
+## Examples: 5432, "127.0.0.1:5432", "localhost:5432"
```
Comment (zmstone) [inline apps/emqx_auth_pgsql/etc/emqx_auth_pgsql.conf]: should quote comma inside ?

File: apps/emqx_bridge_mqtt/etc/emqx_bridge_mqtt.conf [hunk 4]
```diff
@@ -9,8 +9,8 @@
 ## Bridge address: node name for local bridge, host:port for remote.
 ##
 ## Value: String
-## Example: emqx@127.0.0.1,  127.0.0.1:1883
-bridge.mqtt.aws.address = 127.0.0.1:1883
+## Example: emqx@127.0.0.1,  "127.0.0.1:1883"
```
Comment (zmstone) [inline apps/emqx_bridge_mqtt/etc/emqx_bridge_mqtt.conf]: this example seem odd from the source branch.

Comment (zmstone) [reply apps/emqx_bridge_mqtt/etc/emqx_bridge_mqtt.conf]: ah, ok, node names are allowed. quote should be around both names.
i.e. the field is a string, not a string array. ?

File: apps/emqx_auth_mongo/etc/emqx_auth_mongo.conf [hunk 5]
```diff
@@ -17,7 +17,7 @@ auth.mongo.type = single
 ## Value: String
 ##
 ## Examples: 127.0.0.1:27017,127.0.0.2:27017...
```
Comment (zmstone) [inline apps/emqx_auth_mongo/etc/emqx_auth_mongo.conf]: quote example

File: apps/emqx_auth_pgsql/etc/emqx_auth_pgsql.conf (hunk 3 above)
Comment (zmstone) [reply apps/emqx_auth_pgsql/etc/emqx_auth_pgsql.conf]: never mind. the comma is not a part of the value, but to separate examples.

File: .github/workflows/run_cts_tests.yaml [hunk 6]
```diff
@@ -132,10 +132,10 @@ jobs:
...
-          sed -i 's|^[#[:space:]]*auth.mysql.ssl[[:space:]]*=.*|auth.mysql.ssl.enable = on|g' apps/emqx_auth_mysql/etc/emqx_auth_mysql.conf
-          sed -i 's|^[#[:space:]]*auth.mysql.cacertfile[[:space:]]*=.*|auth.mysql.cacertfile = \"/emqx/apps/emqx_auth_mysql/test/emqx_auth_mysql_SUITE_data/ca.pem\"|g' apps/emqx_auth_mysql/etc/emqx_auth_mysql.conf
-          sed -i 's|^[#[:space:]]*auth.mysql.certfile[[:space:]]*=.*|auth.mysql.certfile = \"/emqx/apps/emqx_auth_mysql/test/emqx_auth_mysql_SUITE_data/client-cert.pem\"|g' apps/emqx_auth_mysql/etc/emqx_auth_mysql.conf
-          sed -i 's|^[#[:space:]]*auth.mysql.keyfile[[:space:]]*=.*|auth.mysql.keyfile = \"/emqx/apps/emqx_auth_mysql/test/emqx_auth_mysql_SUITE_data/client-key.pem\"|g' apps/emqx_auth_mysql/etc/emqx_auth_mysql.conf
+          sed -i 's|^[#[:space:]]*auth.mysql.ssl.enable[[:space:]]*=.*|auth.mysql.ssl.enable = on|g' apps/emqx_auth_mysql/etc/emqx_auth_mysql.conf
+          sed -i 's|^[#[:space:]]*auth.mysql.ssl.cacertfile[[:space:]]*=.*|auth.mysql.ssl.cacertfile = /emqx/apps/emqx_auth_mysql/test/emqx_auth_mysql_SUITE_data/ca.pem|g' apps/emqx_auth_mysql/etc/emqx_auth_mysql.conf
```
Comment (zmstone) [inline .github/workflows/run_cts_tests.yaml:136]: why the escaped quotes are gone ?

Comment (z8674558) [reply .github/workflows/run_cts_tests.yaml]: mb fixed

**Review by zmstone — APPROVED** [review]

**Comment by zmstone** [comment]
This PR was mistakenly 'squash' merged.
The commit has been deleted from dev/v5.0 branch with a force push

**Pull request merged**